kpops/components/base_components/kafka_connector.py

```diff
@@ -229,10 +233,12 @@ def apply_from_inputs(self, name: str, topic: FromTopic) -> NoReturn:
 
     @override
     async def reset(self, dry_run: bool) -> None:
+        """Reset the state of a Kafka Connect source connector. The source connector is kept."""
         await self._resetter.reset(dry_run)
 
     @override
     async def clean(self, dry_run: bool) -> None:
+        """Delete the source connector. Reset the state of a Kafka Connect source connector."""
```

Comment:
how about a little more concise: "delete and reset state"? I think it's clear that it's about the connector since that's the component